### ISSUE TITLE
Suggestion: Keyboard shortcut to search all items that can be used in the recipe.

### ISSUE THREAD

**Issue by yqx1110** [mezz/JustEnoughItems#1622]
With this function, you can easily put your cursor on an item, and press a key to search all items that can be used in the recipe. (meaning that they have the same oreDict)

**Comment by mezz** [comment]
Thanks for the suggestion!
Do you mean that you want to look up multiple items at the same time, for instance to see all the recipes that create all types of wool?

**Comment by yqx1110** [comment]
Exactly! It's like when a recipe requires an item with $wool, you can press a key to quickly search all items with $wool.

**Comment by kazimuth** [comment]
I was also going to request this feature :) 

I think it would fix how you sometimes get stuck in loops -- say, clicking on kinds of glass over and over again and just getting "put dye on glass" instead of "put sand in a furnace".

I'd be willing to make a PR if you can point me in the right direction for an implementation.

**Comment by yqx1110** [comment]
> I was also going to request this feature :)
> 
> I think it would fix how you sometimes get stuck in loops -- say, clicking on kinds of glass over and over again and just getting "put dye on glass" instead of "put sand in a furnace".
> 
> I'd be willing to make a PR if you can point me in the right direction for an implementation.

I'm sorry but I don't know how JEI works so I can't really help you. My original thought was when you are looking up a recipe, you can press a key on a slot, it automatically inputs the required item orDict into the search bar and searches it. But there must be a better way.
